packages/gatsby-transformer-remark/.gitignore

```diff
@@ -1,2 +1,3 @@
 /*.js
 !index.js
+utils
```

Comment:
can this be `/utils`? this will gitignore `src/utils` as well

Reply:
done